x-pack/plugins/maps/public/angular/get_initial_layers.test.js

```diff
@@ -26,7 +26,7 @@ describe('Saved object has layer list', () => {
       },
     ];
     const layerListJSON = JSON.stringify(layerListFromSavedObject);
-    expect(getInitialLayers(layerListJSON)).toEqual(layerListFromSavedObject);
+    expect(JSON.parse(layerListJSON)).toEqual(layerListFromSavedObject);
```

Comment:
I'm not sure if we should be changing test-stubs to test stringify and parse are mirrors (??)

probably oversight during debugging, but this basically removed a test and hides a regression